triggers/service/daml-platform.sh

```diff
@@ -29,10 +29,20 @@ bazel run //triggers/service:trigger-service-binary -- \
   --http-port $TRIGGER_SERVICE_HTTP_PORT --ledger-host $LEDGER_HOST --ledger-port $LEDGER_PORT --wall-clock-time &
 TRIGGER_SERVICE_PID=$!
 
-# A smoke test:
-#  curl -X GET \
+# Requests to the trigger service need an authorization header with the JWT token.
+# This can be generated from jwt.io using a payload of the form:
+# {
+#   "https://daml.com/ledger-api": {
+#     "ledgerId": "TRIGGER_SERVICE_TEST",
+#     "applicationId": "TriggerService",
+#     "actAs": ["Alice"]
+#   }
+# }
+# Encode it with the secret "secret" and without base64.
```

Comment:
I don’t know why we keep claiming everywhere that the secret must be `secret` (same for the JSON API). We just use the payload of the JWT. The secret matters for the signature which we do not validate.

Reply:
Ah right... I just copied from the JSON API 😅  I'll change it.